spec/preservation_ingest/complete_ingest_spec.rb

```diff
@@ -17,7 +17,7 @@
     end
 
     it 'updates the accessionWF when no errors are raised' do
-      expect(this_robot.workflow_service).to receive(:update_status)
+      allow(this_robot.workflow_service).to receive(:update_status)
```

Comment:
It's hard to know for some of the `expect` changes in this commit whether they were intended *only* to set a stub or also to verify the stub is called. If we think the former, A-OK. If we think the latter, these changes potentially remove test conditions. What do you think?

Reply:
I looked at each test and left in `have_received` when I felt the test or the test description warranted.

Reply:
I looked over my changes again and added 2 more `expect` lines back in, but I am ok with everything else as is.

Reply:
Works for me, thanks :)